### DISCUSSION TITLE
[PyNE] Como emitir nota fiscal para participantes?

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#92] [Geral]
Algumas participantes solicitaram nota fiscal para solicitar reembolso nas suas empresas. Devo solicitar a nota fiscal seguindo as orientações desse manual https://apyb.python.org.br/guias/como-organizar-eventos/ ?

Há alguma orientação adicional por se tratar de um CPF?

cc @muriloviana @tiidadavena

**Comment by muriloviana** [comment]
@cecivieira pode solicitar através desse formulário sim. Para PF, precisamos do nome completo, CPF, cidade e estado além do email.

**Reply by muriloviana** [reply]
Se possível, adicionar o valor da nota fiscal e se é referente a ingresso inteira ou meia.
